frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -523,11 +527,84 @@ function ShouldUseTreesFormItem() {
   );
 }
 
+function CollapsibleEvaluationSettings({
+  isActive = false,
+  setActive,
+}: { isActive: boolean; setActive: (active: boolean) => void }) {
+  return (
+    <Collapse
+      style={{ marginBottom: 8 }}
+      onChange={() => setActive(!isActive)}
+      expandIcon={() => <Checkbox checked={isActive} />}
+      items={[
+        {
+          key: "evaluation",
+          label: "Evaluation Settings",
+          children: (
+            <Row>
+              <Col style={{ width: "100%" }}>
+                <Form.Item
```

Comment:
This from item with the check box looks a little quirky in vertical mode. Could you please make it horizontal?
```javascript
<Form.Item
  ...
  layout="horizontal"
  />
```

Reply:
Oh I am sorry, I mislooked regarding since what version this property is available. I think I read `4.18` but it is `5.18`. I updated antd a little to support this prop, please pull again and run `yarn install`. This should update the version of antd so that the form items support the `layout` prop. 
Maybe your IDE needs a restart after the update because the TS background process did not register the upgrade and still thinks the code is wrong. At least that's what happened in my case.

Reply:
And you can find the docs about the `layout` prop here: https://ant.design/components/form#formitem. In the version column it clearly says `5.18` (not  `4.18`)